packages/discord.js/src/util/Options.js

```diff
@@ -91,7 +91,7 @@ const { version } = require('../../package.json');
  * only set this if you know what you are doing</warn>
  * @property {BuildStrategyFunction} [buildStrategy] Builds the strategy to use for sharding
  * @property {IdentifyThrottlerFunction} [buildIdentifyThrottler] Builds the identify throttler to use for sharding
- * @property {PresenceData} [presence={}] Presence data to use upon login
+ * @property {PresenceData} [presence] Presence data to use upon login
```

Comment:
We should add `initialPresence` here because that’s what /ws uses and should be the property we suggest as alternative to the deprecated `ClientOptions#presence`.